presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -128,41 +135,29 @@ public void testPublic()
     @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
     public void testTableOwnerPrivileges()
     {
-        String tableName = "alice_table";
-        QueryExecutor aliceExecutor = connectToPresto("alice@presto");
-
-        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
-        aliceExecutor.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
-
         onHive().executeQuery("set role admin;");
-        assertThat(onHive().executeQuery(format("SHOW GRANT USER alice ON TABLE %s", tableName)).
-                project(1, 2, 5, 6, 7, 8, 10)).containsOnly(// Select only the relevant columns of SHOW GRANT
+        assertThat(onHive().executeQuery(format("SHOW GRANT USER %s ON TABLE %s", PRINCIPAL_NAME, tableName)).
+                project(1, 2, 5, 6, 7, 8, 10)).containsOnly(// Skip unknown columns such as "grant_time" of SHOW GRANT
```

Comment:
I understood what you meant. But I rather want to emphasize which columns we exclude, rather than enumerating which columns we include. It's not the case that we are only interested in these particular columns. The fact is, we can't compute values of certain columns and hence are excluding them. I don't see why adding a comment about exclusion is not sufficient.